kie-server-parent/kie-server-wars/kie-server/src/main/assembly/assembly-servlet-container.xml

```diff
@@ -38,6 +38,7 @@
     <dependencySet>
       <excludes>
         <exclude>jakarta.transaction:*</exclude>
+        <exclude>org.jboss.resteasy:resteasy-jaxb-provider</exclude>
```

Comment:
@gmunozfe Could you let me know why this exclusion was required? (It's okay as tests passed, though)

Reply:
Ok, we should keep it, removing this exclusion